components/OverveiwComponents/methods.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Cog } from "lucide-react";
+import Image from "next/image";
+import Link from "next/link";
+
+export const Methods = () => {
+  return (
+    <div className="flex flex-col text-destructive dark:text-secondary">
+      <Cog className="w-[3rem] h-[3rem] " />
+      <h1 className="font-sans text-card-foreground text-2xl md:text-3xl lg:text-4xl font-bold py-[1rem]">
+        {" "}
+        Methods
+      </h1>
+      <p className=" md:text-[1.25rem] text-[1rem] md:w-[45rem] lg:w-[50rem] text-foreground">
+        The presented results were obtained using the PyPSA-Earth power system
+        model , an open-source tool that enables state-of-the-art energy planing
+        for everyone.{" "}
+        <Link
+          href="https://github.com/pypsa-meets-earth/pypsa-earth"
+          target="_blank"
+          className="dark:text-secondary text-destructive"
+        >
+          PyPSA-Earth modeling
+        </Link>{" "}
+        {""} workflow automatically extracts all necessary data from publicly
+        available open-source databases such as OpenStreetMap and
+        GlobalEnegyMonitor. The workflow is designed to be fully functional
+        across the whole world which enables modeling for any counrty of the
+        world.
```

Comment:
A bit more typo-fixing (my bad, sorry for that!):

```suggestion
        {""} workflow automatically extracts all necessary data from publicly
        available open-source databases, such as OpenStreetMap and
        GlobalEnergyMonitor. The workflow is designed to be fully functional
        worldwide, which enables modeling for any country of the
        world.
```